django/db/models/fields/__init__.py

```diff
@@ -239,6 +241,13 @@ def _check_field_name(self):
         else:
             return []
 
+    @property
+    def rel(self):
+        warnings.warn(
+            "Usage of field.rel has been deprecated. Use field.remote_field instead.",
+            RemovedInDjango20Warning, 2)
```

Comment:
I didn't either, I guess you just made a typo with RemovedInDjango20Warning instead of 21.